notifications_utils/template.py

```diff
@@ -704,6 +706,15 @@ def too_many_pages(self):
     def postal_address(self):
         return PostalAddress.from_personalisation(InsensitiveDict(self.values))
 
+    def has_qr_code_with_too_much_data(self):
+        content = self._personalised_content if self.values else self.content
+        try:
+            Take(content).then(notify_letter_qrcode_validator)
```

Comment:
`Take` is only needed if you’re passing the content through multiple functions:
```suggestion
            notify_letter_qrcode_validator(content)
```

Reply:
Yeah, it's redundant; I didn't mind that so much if only for the visual similarity that you end up. I don't think it matters too much either way but I can remove it 🤷

Reply:
(also you end up needing to do `notify_letter_qrcode_validator(str(content))`)